docs/docs/understanding/workflows/observability.md

```diff
@@ -105,7 +105,7 @@ In a notebook environment it can be helpful to run a workflow step by step. You
 
 ```python
 w = ConcurrentFlow(timeout=10, verbose=True)
-handler = w.run()
+handler = w.run(stepwise=True)
 
 while not handler.is_done():
```

Comment:
We should change this snippet to the new way of running step-wise, see https://docs.llamaindex.ai/en/stable/module_guides/workflow/#stepwise-execution

Reply:
Done!